### ISSUE TITLE
Intellisense for scala cells?

### ISSUE THREAD

**Issue by jchapuis** [paiqo/Databricks-VSCode#204]
I can't seem to get intellisense for scala cells, is this supposed to work with the databricks kernel? 🙏

**Comment by gbrueckl** [comment]
No, this is not supposed to work - at least not as part of this extension. 
Using the notebook-feature just allows you to send code (Python, Scala, SQL, ...) to Databricks and show the results.

There might be other extensions that support your intellisense for Scala (or any other language) but this extension has no language specific features.
